### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

**Comment by rnshah9** [comment]
Had to change the linker flag to correctly compile with libFuzzer.

File: demo/makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Address-sanitizer library
```
Comment (xansec) [inline demo/makefile]: Can we set these flags on the command line rather than modifying them in the Makefile?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**